python/pyspark/sql/connect/proto/relations_pb2.pyi

```diff
@@ -1601,3 +1610,48 @@ class RenameColumnsByNameToNameMap(google.protobuf.message.Message):
     ) -> None: ...
 
 global___RenameColumnsByNameToNameMap = RenameColumnsByNameToNameMap
+
+class WithColumns(google.protobuf.message.Message):
+    """Adding columns or replacing the existing columns that has the same names."""
```

Comment:
has -> have

Reply:
this is in fact auto-generated..

Reply:
The comment is auto generated from the comment in the proto.

Reply:
I changed in the source which is `relations.proto`.

Reply:
I pass it text in a new Gmail to find this errors.